### ISSUE TITLE
Error: uploadedFilePath is not defined

### ISSUE THREAD

**Issue by tomclose** [achambers/ember-cli-deploy-original#71]
When trying to deploy assets for the first time, the following happens:

```
$ ember deploy:assets --environment staging
version: 0.1.4
Could not find watchman, falling back to NodeWatcher for file system events
uploadedFilePath is not defined
ReferenceError: uploadedFilePath is not defined
    at /path_omitted/my-app/node_modules/ember-cli-deploy/lib/tasks/deploy-assets.js:61:55
    at $$$internal$$tryCatch (/path_omitted/my-app/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:470:16)
    at $$$internal$$invokeCallback (/path_omitted/my-app/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:482:17)
    at $$$internal$$publish (/path_omitted/my-app/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:453:11)
    at $$$internal$$publishRejection (/path_omitted/my-app/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:395:7)
    at $$rsvp$asap$$flush (/path_omitted/my-app/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1531:9)
    at process._tickDomainCallback (node.js:463:13)
```

**Comment by achambers** [comment]
Thanks Tom. This will be fixed by #45 which will be merged shortly.

**Comment by tomclose** [comment]
I don't think #45 fully fixes this. When I tried that approach the `filePath` still didn't display. I had to do this instead: https://github.com/tomclose/ember-cli-deploy/commit/945806456dc4d4f1bc87a17e3f3132e6a76d2110. (See #72.)
